packages/base/src/index.js

```diff
@@ -4,82 +4,115 @@ module.exports = {
     'shared-node-browser': true,
   },
 
-  plugins: ['import'],
+  plugins: ['import', 'prettier'],
 
-  extends: ['eslint:recommended'],
+  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
 
   rules: {
-    'accessor-pairs': 'error',
-    'array-bracket-spacing': ['error', 'never'],
-    'array-callback-return': 'error',
-    'arrow-parens': 'error',
-    'arrow-spacing': [
+    'prettier/prettier': [
       'error',
       {
-        before: true,
-        after: true,
+        singleQuote: true,
+        trailingComma: 'all',
+        quoteProps: 'consistent',
+      },
+      {
+        // Allow consumers to override this prettier config.
+        usePrettierrc: true,
       },
     ],
-    'block-scoped-var': 'error',
-    'block-spacing': ['error', 'always'],
-    'brace-style': 'error',
-    'camelcase': [
+
+    // In order to match the prettier spec, you have to enable lines around
+    // comments before and after blocks, objects, and arrays.
+    // https://github.com/prettier/eslint-config-prettier#lines-around-comment
+    'lines-around-comment': [
```

Comment:
This had no functional change in the extension, turning it completely off yields 0 code changes. I recommend turning it off or extracting it completely allowing the recommended setup to override the rule to off.